vuepress/docs/next/tutorials/getting-started/kubernetes-install.md

```diff
@@ -10,17 +10,16 @@ Below are the minimum infrastructure requirements to install Entando directly on
 - To install bundles, the cluster containing the Entando Application must have access to the bundle repository (either public or private, often on GitHub) as well as the Docker image repository (e.g. DockerHub) for any microservices.
 - Kubernetes Secrets are required to [install bundles from a private image repository](../curate/private-images.md), such as DockerHub.
 
+The following requirements specify the baseline resource requirements for Entando itself but actual requirements will depend on how Entando is utilized. Use of more features, bundles, and capabilities may increase those requirements. In addition, Kubernetes has its own resource requirements which vary depending on the Kubernetes distribution. At a minimum, configure a single worker node with at least the resources listed below, or designate 3 worker nodes, each with a minimum of 2 vCPUs and 4 GB of RAM, as a starting point.
```

Comment:
slight rewrite: "The baseline resource requirements for Entando are specified below. Actual requirements depend on utilization and increase with features, bundles and capabilities. In addition, Kubernetes has its own resource requirements, which vary depending on the distribution. At a minimum, configure a single worker node with the resources listed below. Alternatively, begin with 3 worker nodes, each with at least 2 vCPUs and 4 GB of RAM."